src/aap_eda/api/views/user.py

```diff
@@ -196,9 +196,10 @@ def perform_create(self, serializer):
         },
     ),
     partial_update=extend_schema(
-        exclude=not settings.ALLOW_LOCAL_RESOURCE_MANAGEMENT,
         description="Partial update of a user.",
-        request=serializers.UserCreateUpdateSerializer,
+        request=serializers.UserCreateUpdateSerializer
```

Comment:
I would prefer a conventional if-else block for readability.

Reply:
already changed. Thanks for catching it.